user-access-manager.php

```diff
@@ -132,9 +132,6 @@ function userAccessManagerAP()
         ) {
             //Admin actions
            if (function_exists('add_action')) {
-                add_action('admin_print_styles', array($oUserAccessManager, 'addStyles'));
-                add_action('wp_print_scripts', array($oUserAccessManager, 'addScripts'));
```

Comment:
This should not be here, although it looks natural to only register and enqueue scripts and styles when the admin is initialized. But nesting hook calls seems to cause trouble:
https://pippinsplugins.com/loading-scripts-correctly-in-the-wordpress-admin/